products/ocp4/profiles/cis-node.profile

```diff
@@ -25,6 +25,8 @@ description: |-
 
     This profile is applicable to OpenShift versions 4.10 and greater.
 
+extends: cis-node-1.4.0
+
 filter_rules: '"ocp4-node" in platforms or "ocp4-master-node" in platforms or "ocp4-node-on-sdn" in platforms or "ocp4-node-on-ovn" in platforms'
 
 selections:
```

Comment:
Remove these selections since they're going to be inherited from the extended profile.